rts/lra/lra-filters/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java

```diff
@@ -351,7 +346,7 @@ public void filter(ContainerRequestContext containerRequestContext) throws IOExc
                     throw new GenericLRAException(lraId, e.getResponse().getStatus(), e.getMessage(), e);
                 }
 
-                headers.putSingle(LRA_HTTP_RECOVERY_HEADER, recoveryUrl);
+                headers.putSingle(LRA_HTTP_RECOVERY_HEADER, recoveryUrl.replaceAll("\"", ""));
```

Comment:
is this safe (from functional point of view) just to remove all `"` parenthesis?